Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM openjdk
+FROM openjdk:8-jdk-alpine
 WORKDIR /
-ADD target/conter-0.0.1-SNAPSHOT.jar conter-0.0.1-SNAPSHOT.jar
+COPY target/conter-0.0.1-SNAPSHOT.jar app.jar
 EXPOSE 8080
-CMD java -jar conter-0.0.1-SNAPSHOT.jar
+ENTRYPOINT [ "sh", "-c", "java -Dutopia.search.API=$1 -Dutopia.booking.API=$2 -Dutopia.cancellation.API=$3 -Djava.security.egd=file:/dev/./urandom -jar /app.jar" ]
```

Comment:
Is `/app.jar` or simply `app.jar` (which is equivalent to `./app.jar`) correct? (Where does the `COPY` directive put the JAR file, in the root of the image or whatever the starting CWD will be?)

Reply:
/ app.jar or app.jar are correct , because I'm copying the jar file to the root of the Image. so it's not a big deal, it's working either ways , I tested it

Reply:
It's not a matter of where it's being copied to, but where in the image the "current working directory" is when it's *run*. But if it works either way, that's fine.